testssl.sh

```diff
@@ -3933,11 +3933,11 @@ sclient_connect_successful() {
 determine_tls_extensions() {
      local proto addcmd
      local success
-     local alpn=""
+     local alpnParam=""
      local savedir
      local nrsaved
 
-     "$HAS_ALPN" && alpn="h2-14,h2-15,h2"
+     "$HAS_ALPN" && alpnParam="-nextprotoneg h2-14,h2-15,h2"
```

Comment:
This is anyway nonsense. ``ALPN=HTTP2``, ``NPN/nextprotoneg=SPDY``

But honestly I need to understand why this code ``openssl s_client  -showcerts -connect HOST:PORT -tlsextdebug -nextprotoneg "<whatever>" -status`` made it into the for loop.